extensions/amp-analytics/amp-analytics.md

```diff
@@ -222,12 +222,24 @@ As an example, the following configuration can be used to sample 50% of the requ
 },
 ```
 
+
+#####  Element selector
+
+Some triggers such as `click` and `visible` allow specifying an single element or a collection of elements using the selector properties. Different triggers can apply different limitations and interpretations on selected elements, such as whether a selector applies to all matched element or the first one, or which elements can be matched: all or only AMP elements. See the documentation of each relevant trigger for more details.
```

Comment:
It's also not clear to me how the selection properties help with "which elements can be matched: all or only AMP elements".